new-dti-website/components/team/TeamHero.tsx

```diff
@@ -161,7 +159,7 @@ const TeamHero = () => {
                     index === carouselIndex % carouselLength ? '' : 'opacity-50'
                   }`}
                 >
-                  <div
+                  <button
```

Comment:
So one issue i found when using `tab` to test these changes is that when you open the image, you cannot use `tab` to navigate through the X close icon and left/right chevron icons. I think when the image is open in this larger view, we should treat it as a modal and so lock the focus to only those 3 elements (X close btn, left chevron, and right chevron). We could also add the ability to use `esc` to close the image, too.

Currently, as you can see in the video, once I open the image in the large view, pressing tab changes the to the next image. But I think we should be reserving the focus state for those 3 elements


https://github.com/user-attachments/assets/84b13a2a-d899-4347-881c-641976ceb685